.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,36 @@
+name: "CodeQL Code Scan"
```

Comment:
```suggestion
name: "CodeQL"
```

That should show the code scan result under `Security` tab?

Reply:
Yep it if is enabled in the security tab, the action will post to the alerts

Reply:
Do we need to click on that in the `Security` tab or do we need to rename it to `CodeQL` ?